packages/fluentui/react-northstar/src/components/Video/Video.tsx

```diff
@@ -24,77 +28,93 @@ export interface VideoProps extends UIComponentProps {
 
   /** Video source URL. */
   src?: string;
-}
-
-export const videoClassName = 'ui-video';
-
-class Video extends UIComponent<WithAsProp<VideoProps>> {
-  static create: ShorthandFactory<VideoProps>;
-
-  static deprecated_className = videoClassName;
-
-  static displayName = 'Video';
-
-  static propTypes = {
-    ...commonPropTypes.createCommon({
-      children: false,
-      content: false,
-    }),
-    autoPlay: PropTypes.bool,
-    controls: PropTypes.bool,
-    loop: PropTypes.bool,
-    muted: PropTypes.bool,
-    poster: PropTypes.string,
-    src: PropTypes.string,
-  };
-
-  static defaultProps = {
-    as: 'video',
-    controls: true,
-    autoPlay: false,
-    muted: false,
-  };
 
-  videoRef = React.createRef<HTMLVideoElement>();
+  accessibility?: Accessibility;
```

Comment:
```suggestion
  accessibility?: Accessibility<VideoBehaviorProps>;
```

Reply:
Also please add description for the prop

Reply:
Done